src/ui/server/controllers/insurance/business/alternative-trading-address/validation/rules/alternative-trading-address.ts

```diff
@@ -0,0 +1,36 @@
+import { ERROR_MESSAGES } from '../../../../../../content-strings';
+import { FIELD_IDS } from '../../../../../../constants';
+import { RequestBody } from '../../../../../../../types';
+import { objectHasProperty } from '../../../../../../helpers/object';
+import generateValidationErrors from '../../../../../../helpers/validation';
+
+const { ALTERNATIVE_TRADING_ADDRESS: FIELD_ID } = FIELD_IDS.INSURANCE.EXPORTER_BUSINESS;
+
+const {
+  EXPORTER_BUSINESS: { [FIELD_ID]: ERROR_MESSAGE },
+} = ERROR_MESSAGES.INSURANCE;
+
+export const MAXIMUM = 1000;
+
+/**
+ * validates alternative trading address input
+ * errors if empty or more than 1000 characters
+ * @param {RequestBody} responseBody
+ * @param {object} errors
+ * @returns {object} errors
+ */
+const alternativeTradingAddress = (responseBody: RequestBody, errors: object) => {
+  // if body is empty
+  if (!objectHasProperty(responseBody, FIELD_ID)) {
+    return generateValidationErrors(FIELD_ID, ERROR_MESSAGE.IS_EMPTY, errors);
+  }
+
+  // check if the field is above the maximum
```

Comment:
Can we use the shared validation function here? `helpers/shared-validation/max-length`, there's an example in `other-comments` validation